sys/riscv/include/frame.h

```diff
@@ -54,6 +54,7 @@ struct trapframe {
 	uintcap_t tf_a[8];
 	uintcap_t tf_sepc;
 	uintcap_t tf_ddc;
+	uintcap_t tf_tidc;
```

Comment:
This doesn't belong in the trapframe, it belongs in the PCB, because it's not clobbered by the kernel unless it performs a context switch.